ui/app/AppLayouts/Wallet/panels/ContractInfoButtonWithMenu.qml

```diff
@@ -0,0 +1,49 @@
+import QtQuick 2.15
```

Comment:
where is this file used? I cant seem to find a usage

Reply:
E.g. here: https://github.com/status-im/status-desktop/pull/15410/files#diff-7e030a701618fd70e713848be11044dd4816d7d9d4900b3b17caffc2cccb5755R159

Reply:
(I renamed the file, so you probably didn't refresh the code)